node/network/protocol/src/lib.rs

```diff
@@ -438,6 +438,62 @@ pub mod v1 {
 	impl_try_from!(CollationProtocol, CollatorProtocol, CollatorProtocolMessage);
 }
 
+/// Unified annoyance cost and good behavior benefits.
```

Comment:
Maybe it is just me, but I really prefer not having everything in a single file. Putting this isolated logic into its separate module would make a lot of sense to me.

Reply:
I agree with you, that more files are better. In this case I chose to put it there, since that is the unique import location of `ReputationChange` from `substrate` and since that has to be in scope, it seemed logical to keep it in that crate. But I can split it into another file.

Reply:
Oh yeah, I would have picked the same crate too. Just a new sub module in that crate, we can even publicly re-export it from lib.